django-backend/fecfiler/transactions/views.py

```diff
@@ -305,6 +305,7 @@ def save_transaction(self, transaction_data, request):
                 schedule_instance.report_coverage_from_date = report.coverage_from_date
                 schedule_instance.save()
 
+        Report.objects.all().update(calculation_status=None)
```

Comment:
Can you limit this to reports owned by the committee?

Reply:
Good catch, got it fixed to only in committee